Rover/mode.cpp

```diff
@@ -97,6 +101,24 @@ void Mode::get_pilot_input(float &steering_out, float &throttle_out)
     }
 }
 
+// roll_out is in the range -100 ~ +100
+// pitch_out is in the range -100 ~ +100
+void Mode::get_pilot_desired_roll_and_pitch(float &roll_out, float &pitch_out)
+{
+    roll_out = rover.channel_roll->get_control_in();
+    pitch_out = rover.channel_pitch->get_control_in();
+
+}
+
+// gait select is in the range -100 ~ +100
+// walking robot mode is in the range -100 ~ +100
+void Mode::get_pilot_desired_gait_and_control_mode(float &gait_select, float &walkingbot_mode)
+{
+    gait_select = rover.channel_gait_select->get_control_in();
+    walkingbot_mode = rover.channel_walkingbot_mode->get_control_in();
```

Comment:
If we are going to have different walking bots mode we should consider an enum.  Using a float is a bit unusual for a "mode".